src/content/en/2021/cdn.md

```diff
@@ -132,27 +133,37 @@ While we observed CDN adoption across different types of content, we will not lo
   )
 }}
 
-Ranking the domains based on their popularity (at the time of this writing sourced from Google CRUX data) in the web and then checking for their CDN usage, the top 1000 websites contribute to the highest usage of CDN. The top websites include domains owned by names like Google, Amazon who contribute to much of the internet traffic we see today and it's no surprise that these names make it to the list of "Top CDN Providers" in the next section. This also backs the fact about the benefits CDN's bring to the table when operating at scale and having the ability to scale further if needed.
+Ranking the domains based on their popularity (at the time of this writing sourced from Google CRUX data) in the web and then checking for their CDN usage, the top 1000 websites contribute to the highest usage of CDN. The top websites include domains owned by names like Google and Amazon who contribute to much of the internet traffic we see today, so it's no surprise that these names make it to the list of "Top CDN Providers" in the [next section](#top-CDN-providers). This also backs the fact about the benefits CDNs bring to the table when operating at scale and having the ability to scale further if needed.
+
+{{ figure_markup(
+  caption="Sites using CDN among top 10 million sites",
```

Comment:
@Navaneeth-akam should we also specify that this is usage of a CDN for **hosting** the site (serving the HTML document) to distinguish from CDN usage of other 1P and 3P resources?

```suggestion
  caption="Percent of websites using a CDN.",
```